app/lib/frontend/templates/package_versions.dart

```diff
@@ -20,6 +20,7 @@ import 'package.dart';
 /// Renders the `views/pkg/versions/index` template.
 String renderPkgVersionsPage(
     Package package,
+    bool isLiked,
```

Comment:
Maybe document that `isLiked == null` implies that the page does not have a like button at all.